build/python/build/libs.py

```diff
@@ -54,8 +54,8 @@
 )
 
 openssl = OpenSSLProject(
-    'https://www.openssl.org/source/openssl-3.0.0-alpha4.tar.gz',
-    'ftp://ftp.cert.dfn.de/pub/tools/net/openssl/source/openssl-3.0.0-alpha4.tar.gz',
+    'https://www.openssl.org/source/old/3.0/openssl-3.0.0-alpha4.tar.gz',
+    'ftp://ftp.cert.dfn.de/pub/tools/net/openssl/source/old/3.0/openssl-3.0.0-alpha4.tar.gz',
```

Comment:
I'd prefer updating to the latest version.

Reply:
i agree but the newest version is not working without adjustments so we should have a working built at first or?

Reply:
the commit-messages have been separated for the two different changes